aecore_eqc/txs_eqc.erl

```diff
@@ -1893,7 +1911,9 @@ gen_ttl() ->
 
 %% Generate a contract
 gen_contract() ->
-    elements(["identity", "authorize_nonce"]).
+    elements([ "identity"
+             %% , "authorize_nonce" TODO Restore `authorize_nonce.aes` alongside `identity.aes` in https://github.com/aeternity/aeternity/tree/master/test/contracts
```

Comment:
should now be in PR

Reply:
I see https://github.com/aeternity/aeternity/pull/2315

Reply:
So I need to adapt this PR where authorize_nonce used.